occurrences/tests/test_api.py

```diff
@@ -1774,3 +1775,75 @@ def test_cancel_occurrence(snapshot, staff_api_client, occurrence):
     for e in occurrence.enrolments.all():
         assert e.status == Enrolment.STATUS_CANCELLED
     assert_match_error_code(executed, API_USAGE_ERROR)
+
+
+ENROLMENTS_SUMMARY_QUERY = """
+query enrolmentSummary($organisationId: ID!, $status: EnrolmentStatus){
+  enrolmentSummary(organisationId: $organisationId, status:$status){
+    totalCount
+    edges{
+      node{
+        status
+      }
+    }
+  }
+}
+"""
+
+
+def test_enrolments_summary_unauthorized(
+    snapshot, api_client, user_api_client, staff_api_client, organisation
+):
+    organisation_gid = to_global_id("OrganisationNode", organisation.id)
+    executed = api_client.execute(
+        ENROLMENTS_SUMMARY_QUERY, variables={"organisationId": organisation_gid}
+    )
+    assert_permission_denied(executed)
+
+    executed = user_api_client.execute(
+        ENROLMENTS_SUMMARY_QUERY, variables={"organisationId": organisation_gid}
+    )
+    assert_permission_denied(executed)
+
+    # assert organisation not in staff_api_client.user.person.organisations
+    executed = staff_api_client.execute(
+        ENROLMENTS_SUMMARY_QUERY, variables={"organisationId": organisation_gid}
+    )
+    assert_permission_denied(executed)
+
+
+def test_enrolments_summary(
+    snapshot, staff_api_client, occurrence, mock_get_event_data
+):
+    organisation_gid = to_global_id(
+        "OrganisationNode", occurrence.p_event.organisation.id
+    )
+    EnrolmentFactory(study_group=StudyGroupFactory(), occurrence=occurrence)
```

Comment:
Just a minor thing of course, but using `StudyGroupFactory` here seems unnecessary as that should happen by default.

Reply:
Fixed, it was `Enrolment.objects.create` before I realized there is already EnrolmentFactory. Then I didn't bother to refactor the argument xD